README.md

```diff
@@ -7,3 +7,38 @@ To learn more, please see https://vip.wordpress.com/documentation/vip-go/underst
 ## Usage
 
 All the directories here are required and will be available on production web servers. Any extra directories will not be available in production.
+
+## PHPCS
```

Comment:
This content could be moved on to a `https://vip.wordpress.com/documentation/vip-go/...` page, but I thought it's worth adding it here as a start. cc @tessaneedham

Reply:
Let's get a link to https://docs.wpvip.com/how-tos/php_codesniffer/ and any other relevant pages in that now exist.